mobile/go.mod

```diff
@@ -217,4 +217,4 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // replace can be removed in favour of importing that.
 replace github.com/lightninglabs/faraday => github.com/lightninglabs/faraday v0.2.13-alpha.0.20241118202659-a3aba5b7ea49
 
-go 1.22.6
+go 1.24.2
```

Comment:
Leaving a comment here for this file in general:
We should update the dependencies in this file similar to how you've updated them in the wasm client, as we need both of them to use the same dependencies during a release. But, feel free to ignore this feedback though, as I'll fix it during the release PR if you do :)

Reply:
Good to know. I think for context sake it probably makes sense to do that in the release PR.

I'm not sure how you'd like to deal with the `github.com/lightninglabs/lightning-node-connect v0.3.3-alpha.0.20250303090341-f823ad04b9de` dependency for example now that wasm-client is using relative path to LNC modules.

And I'm not sure if this TODO can be removed:
```
// TODO: uncomment once LiT has been updated to point to the latest version of
// LNC and LND.
// replace github.com/lightninglabs/lightning-node-connect => ../../
```

Reply:
Ok thanks, I'll look into that for the release PR instead!

> // TODO: uncomment once LiT has been updated to point to the latest version of
// LNC and LND.
// replace github.com/lightninglabs/lightning-node-connect => ../../

No, we intentionally kept this, as this was recently added. See this comment for background to why we have this TODO:
https://github.com/lightninglabs/lightning-node-connect/pull/112#discussion_r1981747530